python/ray/tune/schedulers/pbt.py

```diff
@@ -183,6 +184,17 @@ def __init__(self,
                  resample_probability=0.25,
                  custom_explore_fn=None,
                  log_config=True):
+        for value in hyperparam_mutations.values():
+            if not (isinstance(value, list) or callable(value)):
+                raise TypeError(
+                    "hyperparam_mutation values must be either "
+                    "lists or functions")
+            if callable(value) and len(inspect.getfullargspec(value)[0]) != 0:
```

Comment:
This actually doesn't work on `py2`. 
```
E           AttributeError: 'module' object has no attribute 'getfullargspec'
```
https://stackoverflow.com/questions/27012248/python-2-7-getfullargspec-equivalent